app/controllers/admin/people_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Admin
```

Comment:
I am :+1: for testing the controller action independently from adding markup to the view.